samples/subsys/usb/dfu/sample.yaml

```diff
@@ -2,6 +2,7 @@ sample:
   name: USB DFU sample
 tests:
   sample.subsys.usb.dfu:
+    build_only: True
```

Comment:
nit-pick: s/True/true, it is `build_only: true` everywhere in samples and tests.